scripts/update-golden-files.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
A short explanation of what the script is meant for would be useful.

Reply:
Can you check again @simonpasquier wdyt?